client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/PersonalInfoTab.tsx

```diff
@@ -0,0 +1,342 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { FormControl } from '@material-ui/core';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { Typography, RadioGroup, Radio, Collapse } from '@material-ui/core';
+
+import Gender from 'models/enums/Gender';
+import Toggle from 'commons/Toggle/Toggle';
+import CircleSelect from 'commons/CircleSelect/CircleSelect';
+import CustomCheckbox from 'commons/CheckBox/CustomCheckbox';
+import RelevantOccupations from 'models/enums/RelevantOccupations'
+import CircleTextField from 'commons/CircleTextField/CircleTextField';
+import PersonalInfoDataContextFields from 'models/enums/PersonalInfoDataContextFields';
+import { personalInfoContext } from 'commons/Contexts/PersonalInfoStateContext';
+
+import useStyles from './PersonalInfoTabStyles';
+import IdentificationTypes from 'models/enums/IdentificationTypes';
+
+const PersonalInfoTab: React.FC = (): JSX.Element => {
+    const classes = useStyles({});
+
+    const personalInfoStateContext = React.useContext(personalInfoContext);
+
+    const handleChangeField = (fieldName: PersonalInfoDataContextFields, fieldValue: any) => {
+        personalInfoStateContext.setPersonalInfoData({...personalInfoStateContext.personalInfoData, [fieldName]: fieldValue})
+    }
+
+    return (
+        <div className={classes.tabInitialContainer}>
+            <Grid container spacing={3} className={classes.containerGrid} alignItems='center'>
+                <Grid item xs={2} className={classes.PersonalInfoFieldContainer}>
+                    <Typography className={classes.fontSize15}>
+                        <b>
+                            טלפון:
```

Comment:
this string repeats itself, you can put this in a const

Reply:
i think you should put all the strings you're using in consts, the component will be more readable